src/SceneGenerator.js

```diff
@@ -5,7 +5,7 @@ var SceneGenerator = function(filterName, width, height) {
   this.height = height;
   this.filter = filterName;
   this.position.setTo(game.world.width*1.25,game.world.height*0.25);
-  this.texture = game.cache.getBitmapData('randomTex').texture;
+  this.texture = PIXI.Texture.fromCanvas(game.cache.getCanvas('randomTex'));
```

Comment:
PIXI is the renderer that Phaser is built off of. Most of Phaser's functionality is as a game engine, if we want to access any of the serious renderer capabilities we have to access PIXI directly.